lib/ansible/modules/uri.py

```diff
@@ -107,12 +107,14 @@
     default: no
   follow_redirects:
     description:
-      - Whether or not the URI module should follow redirects. V(all) will follow all redirects.
-        V(safe) will follow only "safe" redirects, where "safe" means that the client is only
-        doing a GET or HEAD on the URI to which it is being redirected. V(none) will not follow
-        any redirects. Note that V(true) and V(false) choices are accepted for backwards compatibility,
-        where V(true) is the equivalent of V(all) and V(false) is the equivalent of V(safe). V(true) and V(false)
-        are deprecated and will be removed in some future version of Ansible.
+      - Whether or not the URI module should follow redirects.
+      - V(all) will follow all redirects.
+      - V(safe) will follow only "safe" redirects, where "safe" means that the client is only
+        doing a GET or HEAD on the URI to which it is being redirected.
+      - V(none) will not follow any redirects.
+      - Note that V(true) and V(false) choices are accepted for backwards compatibility,
+        where V(true) is the equivalent of V(all) and V(false) is the equivalent of V(safe).
+        V(true) and V(false) are deprecated and will be removed in Ansible 2.22.
```

Comment:
its confusing to use true/false here when choices are explicitly yes/no ... im guessing this was a 'fix' from those 'standarizing' booleans. It should reflect yes/no as the field is a string and not a boolean.